apps/champions/lib/champions/users.ex

```diff
@@ -88,4 +88,34 @@ defmodule Champions.Users do
     Currencies.add_currency(user.id, Currencies.get_currency_by_name!("Gold").id, 100)
     Currencies.add_currency(user.id, Currencies.get_currency_by_name!("Gems").id, 500)
   end
+
+  @doc """
+  Adds the given experience to a user. If the user were to have enough resulting experience to level up,
+  it is performed automatically.
+  """
```

Comment:
Just a personal comment but i think examples of the function usage are really helpful for example this method receives an user_id and a amount of experience and returns either `{:ok, %User{}}` of `{:error, %Changeset{}}`

Reply:
Yeah I'm not too much of a fan of examples since they take up a lot of space in the code, I like specs more since they accomplish the same in much less space, but we don't have them in this project :(. I try to just specify the return types in the doc but I missed this one heh